doc/manual/manual.tex

```diff
@@ -9937,6 +9937,67 @@ \subsubsection{Benchmarks for operator splitting}
 
 The results shown here can be reproduced using the bash scripts \texttt{run.sh} in the corresponding benchmark folders. 
 
+
+
+
+\subsubsection{Donea \& Huerta 2D box geometry benchmark}
+\label{sec:benchmark-donea-huerta}
+
+\textit{This section was contributed by Cedric Thieulot.}
+
+This benchmark is taken from Donea and Huerta's book \cite{DH03book}.
+The domain is a unit square and the viscosity and density are set to 1. 
+The components of the gravity vector $\mathbf g$ are prescribed as 
+\begin{eqnarray}
+g_x &=& (12 - 24y) x^4 + (-24 + 48y) x^3 + (-48y + 72y^2 - 48 y^3 + 12) x^2 \nonumber\\
+    && + (-2 + 24y -72y^2+48y^3)x + 1-4y + 12y^2-8y^3 \nonumber\\ 
+g_y &=& (8 - 48y + 48 y^2) x^3 + (-12 + 72y - 72y^2) x^2  \nonumber\\
+    && + (4 - 24y + 48y^2 - 48y^3 + 24y^4) x - 12y^2 + 24y^3 - 12y^4. 
+\end{eqnarray}
+The exact solution is then
```

Comment:
maybe say "The exact solution can then be chosen as follows, if one prescribes Dirichlet boundary values for the velocity using the same formula:"

Reply:
agreed, done.

Reply:
Nope, not done :-)